.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,29 +3,57 @@ name: Bug report
 about: Create a report to help us improve
 
 ---
+<!-- Please follow the template below -->
+
+# Describe the bug
 
-**Describe the bug**  
 A clear and concise description of what the bug is.
 
-**To Reproduce**  
+
+# Steps To Reproduce
+
 Steps to reproduce the behavior:
 1. Make a player with a gun, magazine, and some ammo.
 2. Try to aim at a wall.
 3. There's no way to increase aim level.
 
-**Expected behavior**  
-A clear and concise description of what you expected to happen.  
+
+# Expected behavior
+
+A clear and concise description of what you expected to happen.
 Ideally also describe *why* you expect it to happen.
 
-**Screenshots**  
-If applicable, add screenshots to help explain your problem.
 
-**Versions and configuration(please complete the following information):**
+# Screenshots
+
+**If applicable**, add screenshots to help explain your problem.
+
+<!-- Do not take a screenshot of the crashing window. Please provide the crash logs in the "Additional context" -->
+
```

Comment:
```suggestion
```

Reply:
Thanks a lot for your review! I'm applying your changes.